internal/proto/local/plugin/host.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package plugin.sdk.proto;
+
+option go_package = "github.com/hashicorp/boundary/internal/plugin/sdk/proto;proto";
+
+import "google/protobuf/struct.proto";
+
+// Host contains all fields related to a host returned from a plugin.
+message Host {
+  // The primary address for the host result.
+  string address = 10;
+
+	// Name of the host.
+	string name = 20;
+
+	// The host catalog of which this host is a part.
+	string host_catalog_id = 30;
```

Comment:
All requests to the plugins currently will only ever include a single host catalog.  Do we want to require the plugin to populate this data or can we let the controller derive it based on the request?

Reply:
Yeah, given the discussion on the member set field, just going to remove this too.